jdtls.ext/com.microsoft.jdtls.ext.core/src/com/microsoft/jdtls/ext/core/PackageCommand.java

```diff
@@ -311,6 +320,7 @@ private static List<PackageNode> getPackageFragmentRoots(PackageParams query, IP
 
                 for (IPackageFragmentRoot fragmentRoot : packageFragmentRoots) {
                     children.add(fragmentRoot);
+                    children.addAll(Arrays.asList(fragmentRoot.getNonJavaResources()));
```

Comment:
the `nonJavaResources` should be under the fragment root node or do some filtering? otherwise we will get a lot of META-INFs in the Project and Dependencies node.

![image](https://user-images.githubusercontent.com/45906942/230817233-090e410f-d880-4eb9-8698-6553598640ab.png)

Reply:
Good finding!

I should not call `getNonJavaResources()` here. Here we just need to get all the jar fragments as the children of the container node. Those `META-INF` folders belong to the jars and should be shown later when user expand the jar node.